src/aliasRegex.js

```diff
@@ -1,8 +1,24 @@
-export const allowedAliasCharacters = "\\w\\-\\_\\+\\*\\(\\)\\!#&åô’çéãí“”,";
-export const startOfURL = "https?\\S*";
+import asciiAliases from "../data/asciiAliases";
+import flatten from "lodash.flatten";
+
+import { escapeStringToBeUsedInRegExp } from "./utils";
+
+const allowedAliasCharacters = "\\w\\-\\_\\+\\*\\(\\)\\!#&åô’çéãí“”,";
+const startOfURL = "https?\\S*";
+
+const names = flatten(
+  Object.keys(asciiAliases).map(name => {
+    return asciiAliases[name].map(escapeStringToBeUsedInRegExp);
+  })
+).sort().reverse().join("|");
```

Comment:
Just curious, how sorting and reversing names changes anything on regex matching of a group ?

Reply:
To ensure matching the most specific alternative. For example:

```js
> /(a|ab)/.exec("ab")[0]
'a'
> /(ab|a)/.exec("ab")[0]
'ab'
```

It's not actually a problem at the moment (I checked), but best practice seems to me to always reverse sort when generating a group of alternatives to prevent bugs later on.

Reply:
Indeed 👌